docs/contributing-guide/6. submitting-a-pr.md

```diff
@@ -12,35 +12,37 @@
 
 ### Prerequisites to publishing a Pull Request
 
-Once you have gotten setup with the repo, made and tested your changes, there are two more small checks before submitting your PR:
+Once you have gotten setup with the repo, made and tested your changes, there are some checks before submitting your PR:
 
-1. Update any `API` files
-1. Generate required change files
+1. Update dependencies lock file
+2. Update package API files
+3. Generate required change files
 
-#### 0. Update dependencies lock file for all flavors
+#### 1. Update dependencies lock file for all flavors
 If you are adding/updating dependencies in any package.json file, you will need to update dependencies lock files for both beta and stable flavors before submitting changes, run these commands to ensure they are correctly generated:
 
 For beta:
-
-`rush update`
+```bash
+rush update
+```
 
 For stable:
+```bash
+rush update:stable
+```
+These commands will update your dependencies locally and generate pnpm-lock.yaml. You might wanna install the right dependencies for your current flavor (by default, run `rush switch-flavor:beta` again for switching back to beta flavor).
 
-`rush update:stable`
-
-These commands will update your dependencies locally and generate pnpm-lock.yaml. You might wanna install the right dependencies for your current flavor (by default, run `rush switch-flavor:beta` again for switching back to beta flavor)
-
-#### 1. Update package API files
+#### 2. Update package API files
 
 To ensure we do not break any existing public APIs, we use [api-extractor](../infrastructure/api-extractor.md) across all our npm packages. This generates a file detailing all public exports of the package. You can find this file under `package-root/review/beta/package-name.api.md`. We do this to prevent any accidental breaking changes to the packages we export, and to ensure we do not accidentally publish any internal helper classes/functions.
 
 When a package is built the api-extractor is automatically run and will update the corresponding `api.md` file.
 
-- If you are updating apis under stable version, api update for both beta&stable would be necessary, a quick command to do so is `rush build:all-flavors`, which will build under all flavors and generate api files for both. Or you could also switch to stable flavor by running `rush switch-flavor:stable` and do the step below to generate stable api.md (don't forget to run `rush switch-flavor:beta` when finished if necessary)
+- If you are updating apis under stable version, api update for both beta&stable would be necessary, a quick command to do so is `rush build:all-flavors`, which will build under all flavors and generate api files for both. Or you could also switch to stable flavor by running `rush switch-flavor:stable` and do the step below to generate stable api.md (don't forget to run `rush switch-flavor:beta` when finished if necessary).
```

Comment:
```suggestion
- If you are updating APIs for the stable (released) version, API updates for both beta&stable is necessary. A quick command to do so is `rush build:all-flavors`, which will build all flavors and generate API files for both. You could also switch to the stable flavor by running `rush switch-flavor:stable` and perform the step below to generate stable api.md (don't forget to run `rush switch-flavor:beta` when finished if necessary).
```

Reply:
Is `stable` == `release` version? If I remember correct, we have `beta release`(https://github.com/Azure/communication-ui-library/releases) version over night